service/controller/cluster.go

```diff
@@ -118,10 +118,12 @@ func newClusterResourceSets(config ClusterConfig) ([]*controller.ResourceSet, er
 			NTPServers:         config.NTPServers,
 			ProjectName:        config.ProjectName,
 			OIDC: cloudconfig.OIDCConfig{
```

Comment:
You will need to re-generate the golden files in these unit tests to make this build.

Reply:
Sure :) 
How can I do that?

Reply:
Actually that doesn't seem to be the issue - let me check. It builds for me locally

Reply:
:+1:

Reply:
@giantswarm/team-ludacris Can someone take a look here? I can not figure out why this build is failing.

Reply:
Maybe the CircleCI runner is OOM? https://forum.golangbridge.org/t/go-build-exits-with-signal-killed/513

Reply:
I had this repeatedly with kvm-operator too, intermittently. Retrying the build worked, not sure what the root cause is unless it is OOM or something Circle-side (my failures were right after a documented Circle outage so I guessed it was related)

Reply:
It actually passed after a re-run